### DISCUSSION TITLE
Ability to start creating a pattern from the context menu

### DISCUSSION THREAD

**Discussion by x9136** [Chatterino/chatterino2#5205] [Ideas]
Since messages are often the source of patterns, it would be handy to have an option to create a pattern from the context menu.

RMB on a message → Create a pattern... → The pattern table opens with the entered message and the highlighted line → Refine the pattern if necessary → Ok

**Comment by pajlada** [comment]
This could be handy for quick-creating highlights / ignores / filters, so when created it should take that into consideration when making the menu entry.
